core/shared/src/main/scala/org/http4s/Message.scala

```diff
@@ -53,6 +53,8 @@ sealed trait Message[F[_]] extends Media[F] { self =>
 
   def attributes: Vault
 
+  def normalize: Self
```

Comment:
Since this would be a no-op on Response, should we only define this on Request? What is the advantage of having it on the super type?

Reply:
Right, that was a fairly random decision - was mostly looking for some feedback on the approach - see last paragraph of the PR description.

Reply:
@isomarcte Ah yes, I removed most of the description because it was outdated - visible under "edited" still if you are interested :)